src/borg/repoobj.py

```diff
@@ -29,9 +30,9 @@ def format(
         meta: dict,
         data: bytes,
         compress: bool = True,
-        size: int = None,
-        ctype: int = None,
-        clevel: int = None,
+        size: Optional[int] = None,
+        ctype: Optional[int] = None,
+        clevel: Optional[int] = None,
```

Comment:
why did you change this?

Reply:
This is the correct way of representing types which can be None. So after looking into this code, I thought to correct this. Though in the later part we assert that they are not None based on conditions. Maybe I should put this into a different PR.

Reply:
Yeah, if this is not required to be here, a separate types cleanup PR for repoobj.py would be welcome.